packages/webapp/src/views/land/check-land-boundary-details.html

```diff
@@ -23,48 +23,63 @@
                         rows: [
                             {
                                 key: {
-                                text: 'Land boundary file uploaded'
-                            },
+                                    text: 'Land boundary file uploaded'
+                                },
                                 value: {
-                                text: landBoundaryFileName
-                            },
+                                    text: landBoundaryFileName
+                                },
                                 actions: {
                                 items: [
                                     {
                                         href: '/land/check-land-boundary-file',
-                                        text: 'Change'
+                                        text: 'Change',
+                                        attributes: {
+                                            id: "check-land-boundary-file"
```

Comment:
If these IDs are meant to be included in this pull request please add the ticket ID to the PR description and title. As they're not on BNGP-1099